pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -475,8 +494,11 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListeners(env model.En
 				newFilterChains = append(newFilterChains, mutable.Listener.FilterChains...)
 				currentListener.FilterChains = newFilterChains
 			} else {
-				listenerMap[listenerMapKey] = mutable.Listener
-				listenerTypeMap[listenerMapKey] = servicePort.Protocol
+				listenerMap[listenerMapKey] = &listenerEntry{
+					creationTime: service.CreationTime,
```

Comment:
remove this.. this is not useful anyway, because a listner is shared by many services.

Reply:
done.